blueocean-rest/src/main/java/io/jenkins/blueocean/rest/pageable/PagedResponse.java

```diff
@@ -29,7 +29,6 @@
 @Target(METHOD)
 @InterceptorAnnotation(PagedResponse.Processor.class)
 public @interface PagedResponse {
-    public static final int DEFAULT_LIMIT=100;
```

Comment:
This can break backward compatibility. However quick [search](https://github.com/search?q=org%3Ajenkinsci+PagedResponse.+DEFAULT_LIMIT&type=Code&utf8=✓)  shows its used only internally.